rero/rero-ils-ui
# general: use ngVarDirective from @rero/ng-core

The NgVarDirective has moved to `ng-core` project. This commit updates
the import references to this directive.

Co-Authored-by: Renaud Michotte <renaud.michotte@gmail.com>

## Dependencies

Should be merge after https://github.com/rero/ng-core/pull/319 integration and new release of ng-core.

* https://github.com/rero/ng-core/pull/319

## How to test?

* `ng build shared`
* run application in localhost
* Go to "persons" search view 
* Check that birth date and/or death date are displayed for person.

## Code review check list
